_template/alert-policies/example-alert-policy/static-alert.yml

```diff
@@ -22,7 +22,7 @@ valueFunction: SINGLE_VALUE
 terms:
   - priority: CRITICAL
     # Operator used to compare against the threshold.
-    operator: ABOVE | BELOW | EQUALS
+    operator: ABOVE | ABOVE_OR_EQUALS | BELOW | BELOW_OR_EQUALS | EQUALS | NOT_EQUALS
```

Comment:
on the associated value for the `WARNING` threshold below, we should trim the `operator` and `thresholdOccurences` fields to a single value similar to the `baseline-alert.yml` template